### PR TITLE
Add psra sql copy fix

### PR DESCRIPTION
**Pull request by drotheram**
Major updates to automated earthquake risk indicator generation.

- Harmonize with major changes to input data in PSRA and ancillary datasets
- removal of all sql 'COPY' statements and replaced with meta '\copy' command 
- changes to the way point geometries are stored in ElasticSearch

### CODE REVIEW

File: python/add_data.sh [hunk 1]
```diff
@@ -259,105 +507,68 @@ do
     fi
 done 
 
+############################################################################################
+#######################     Import Data from PostGIS to ElasticSearch   ####################
```
Comment (jvanulde) [inline python/add_data.sh:511]: @anthonyfok we will need to figure out how to write the pygeoapi configuration dynamically based on the configuration.

File: python/psra_postgres2es.py [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+#Main Function
+def main():
+    logFileName = '{}.log'.format(os.path.splitext(sys.argv[0])[0])
+    logging.basicConfig(level=logging.INFO,
+                        format='%(asctime)s - %(levelname)s - %(message)s',
+                        handlers=[logging.FileHandler(logFileName),
+                                  logging.StreamHandler()])
+    auth = get_config_params('config.ini')
+    args = parse_args()
+    view = "psra_{province}_{dbview}_{idField}".format(**{
```
Comment (jvanulde) [inline python/psra_postgres2es.py:33]: @anthonyfok we may need a test to make sure that the view contains these fields. Please consult with @drotheram.

File: sample.env [hunk 3]
```diff
@@ -2,8 +2,10 @@ POSTGRES_USER=postgres
 POSTGRES_PASS=password
 POSTGRES_PORT=5432
 DB_NAME=opendrr
+POPULATE_DB=0
```
Comment (jvanulde) [inline sample.env:5]: We should what this parameter does.

**Review by jvanulde — APPROVED** [review]
The ES index mappings look different across the input types. Do we need to update them? Other than that, it looks good to me.

**Review by jvanulde — APPROVED** [review]

File: python/add_data.sh [hunk 4]
```diff
@@ -189,6 +124,7 @@ curl -o social-vulnerability-index.csv \
 
 psql -h db-opendrr -U ${POSTGRES_USER} -d ${DB_NAME} -a -f Create_table_sovi_index_canada_v2.sql
 psql -h db-opendrr -U ${POSTGRES_USER} -d ${DB_NAME} -a -f Create_table_sovi_census_canada.sql
+#psql -h db-opendrr -U ${POSTGRES_USER} -d ${DB_NAME} -a -f Create_table_sovi_thresholds.sql
```
Comment (wkhchow) [inline python/add_data.sh:127]: will be used for next revision

**Review by wkhchow — APPROVED** [review]
add_data.sh sql script orders looks good.

**Review by jvanulde — APPROVED** [review]

**Pull request merged**